src/Symfony/Component/Yaml/Tests/DumperTest.php

```diff
@@ -99,7 +99,7 @@ public function testSpecifications()
                 } elseif (!empty($test['todo'])) {
                     // TODO
                 } else {
-                    eval('$expected = '.trim($test['php']).';');
```

Comment:
why removing the trim` ?

Reply:
It should not make a difference, but I am re-adding it now.